### PR TITLE
Update Algorithm Factorialize A Number

### PR DESCRIPTION
**Pull request by koustuvsinha**
Refer #1094

### CODE REVIEW

File: Algorithm-Factorialize-A-Number.md [hunk 1]
```diff
@@ -73,29 +83,21 @@ factorialize(5);
```
Comment (atjonathan) [inline Algorithm-Factorialize-A-Number.md]: Add Relevant Links and Code Explanation

File: Algorithm-Factorialize-A-Number.md [hunk 2]
```diff
@@ -73,29 +83,21 @@ factorialize(5);
...
+### Relevant Links
 
 - **Recursion**: [https://www.codecademy.com/courses/javascript-lesson-205/0/1](https://www.codecademy.com/es/courses/javascript-lesson-205/0/1)
 - **Factorialization**: [https://en.wikipedia.org/wiki/Factorial](https://en.wikipedia.org/wiki/Factorial)
 - **Arithmetic Operators** [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators)
 
-## Credits:
+## :trophy: Credits:
 
 If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`Thanks @luishendrix92 @Rafase282 @hallaathrad for your help with Algorithm: Factorialize a Number`**
```
Comment (atjonathan) [inline Algorithm-Factorialize-A-Number.md]: Add two lines between `on the main chat:` and `Thanks`:
```diff
- If you found this page useful, you can give thanks by copying and pasting this on the main chat: **`Thanks @luishendrix92 @Rafase282 @hallaathrad for your help with Algorithm: Factorialize a Number`**
+ If you found this page useful, you can give thanks by copying and pasting this on the main chat:
+
+ **`Thanks @luishendrix92 @Rafase282 @hallaathrad for your help with Algorithm: Factorialize a Number`**
```

File: Algorithm-Factorialize-A-Number.md [hunk 3]
```diff
@@ -73,29 +83,21 @@ factorialize(5);
 
 :rocket: [Run Code](https://repl.it/CLjU/1)
 
-## References
+### Relevant Links
 
 - **Recursion**: [https://www.codecademy.com/courses/javascript-lesson-205/0/1](https://www.codecademy.com/es/courses/javascript-lesson-205/0/1)
```
Comment (atjonathan) [inline Algorithm-Factorialize-A-Number.md]: ~~`**Recursion**: [https://www.codecademy.com/courses/javascript-lesson-205/0/1](https://www.codecademy.com/es/courses/javascript-lesson-205/0/1)`~~ `[Recursion](https://www.codecademy.com/es/courses/javascript-lesson-205/0/1)`

File: Algorithm-Factorialize-A-Number.md [hunk 4]
```diff
@@ -73,29 +83,21 @@ factorialize(5);
...
 - **Factorialization**: [https://en.wikipedia.org/wiki/Factorial](https://en.wikipedia.org/wiki/Factorial)
```
Comment (atjonathan) [inline Algorithm-Factorialize-A-Number.md]: ~~`**Factorialization**: [https://en.wikipedia.org/wiki/Factorial](https://en.wikipedia.org/wiki/Factorial)`~~ `[Factorialization](https://en.wikipedia.org/wiki/Factorial)`

File: Algorithm-Factorialize-A-Number.md [hunk 5]
```diff
@@ -73,29 +83,21 @@ factorialize(5);
...
 - **Arithmetic Operators** [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators)
```
Comment (atjonathan) [inline Algorithm-Factorialize-A-Number.md]: ~~`**Arithmetic Operators** [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators)`~~ `[Arithmetic Operators](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Arithmetic_Operators)`

**Comment by Rafase282** [comment]
1. Add the image to the top.
2. Move the explanations to the hints section.

File: Algorithm-Factorialize-A-Number.md [hunk 6]
```diff
@@ -73,29 +83,27 @@ factorialize(5);
 
 :rocket: [Run Code](https://repl.it/CLjU/1)
 
-## References
+## Code Explanation
```
Comment (atjonathan) [inline Algorithm-Factorialize-A-Number.md]: ~~Code Explanation~~ Code Explanation:

File: Algorithm-Factorialize-A-Number.md [hunk 7]
```diff
@@ -1,14 +1,18 @@
 # Algorithm Factorialize A Number
 
-## Problem Explanation:
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
```
Comment (Rafase282) [inline Algorithm-Factorialize-A-Number.md]:
```diff
-:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
-
-![Recursion](http://i61.tinypic.com/28auvsw.jpg)
+![Recursion](http://i61.tinypic.com/28auvsw.jpg)
+:triangular_flag_on_post: Remember to use [**`Read-Search-Ask`**](FreeCodeCamp-Get-Help) if you get stuck. Try to pair program :busts_in_silhouette: and write your own code :pencil:
```

File: Algorithm-Factorialize-A-Number.md [hunk 8]
```diff
@@ -1,14 +1,18 @@
...
+## :checkered_flag: Problem Explanation:
 
 Return the factorial of the provided integer. If the integer is represented with the letter n, a factorial is the product of all positive integers less than or equal to n.
 
 Factorials are often represented with the shorthand notation n!
 
 For example: `5! = 1 * 2 * 3 * 4 * 5 = 120`
 
-## Outline
+## Hints
```
Comment (Rafase282) [inline Algorithm-Factorialize-A-Number.md]:
```diff
-## Hints
+## :speech_balloon: Hint: 1
```

Comment (Rafase282) [inline Algorithm-Factorialize-A-Number.md]: Change the other to that too, you can add the previous title in bold underneath.

**Comment by Rafase282** [comment]
https://github.com/koustuvsinha/wiki/pull/2

**Comment by Rafase282** [comment]
# LGTM

**Pull request merged**